src/components/loader.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import pageLoader from 'images/load.gif';
+import searchLoader from 'images/Spinner.gif';
+
+export function PageLoader ( {state} ) {
```

Comment:
To simplify the PageLoader component:
Page loader shouldn't receive any props. Because you are already rendering it (or not) in the component where it's called. Depending on the state of that component.